test/css/value/unicode.0.json

```diff
@@ -0,0 +1,92 @@
+{
+  "type": "value",
+  "content": [
+    {
+      "type": "ident",
+      "content": "U",
```

Comment:
I'm asking because `-` and `+` in this case are not operators and `U+0-7F` is a single unit that in my opinion should not be broken down into smaller parts.